apps-client/backoffice/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@client/backoffice",
+  "type": "module",
+  "scripts": {
+    "build": "react-router build",
+    "dev": "react-router dev",
+    "start": "react-router-serve ./build/server/index.js",
+    "lint": "eslint . --ext .ts,.tsx",
+    "format": "prettier --write .",
+    "format:check": "prettier --check .",
+    "generate": "react-router typegen",
+    "typecheck": "react-router typegen && tsc"
+  },
+  "dependencies": {
+    "@react-router/fs-routes": "^7.6.3",
+    "@react-router/node": "^7.5.3",
+    "@react-router/serve": "^7.5.3",
+    "isbot": "^5.1.27",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
+    "react-router": "^7.5.3"
+  },
+  "devDependencies": {
+    "@pple-today/project-config": "workspace:^",
+    "@pple-today/ui": "workspace:^",
+    "@react-router/dev": "^7.5.3",
+    "@tailwindcss/vite": "^4.1.4",
+    "@types/node": "^20",
+    "@types/react": "~19.0.10",
+    "@types/react-dom": "19.0.0",
+    "tailwindcss": "^4.1.4",
```

Comment:
not sure if tailwind 4 is a good fit for now
note that we have nativewind ~ tailwindv3 in react native and may support v4 very soon 
https://github.com/nativewind/nativewind/discussions/1422